tests/integration/datadict/submission/test_bcr_xml_to_json.py

```diff
@@ -18,7 +18,7 @@ def test_gdc_type_mappings():
 
     # parse long
     long_val = munge_property("3111118", "long")
-    assert isinstance(long_val, long)
+    assert isinstance(long_val, int)
```

Comment:
this part of the test doesn't really make sense anymore since it's testing `int` a second time

Reply:
yeah it is but I think we can leave it because we want to make sure even if someone is passing `long` as input to `munge_property` it is still returning an int?

Reply:
sure :D